sql/sql-optimizer-overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: SQL Optimization Process
+summary: Learn about the logical and physical optimization of SQL in TiDB.
+category: user guide
+---
+
+# SQL Optimization Process
+
+In TiDB, the process of SQL optimization consists of two phases: logical optimization and physical optimization. This document describes the logical and physical optimization to help you understand the whole process.
+
+## Logical optimization
+
+Based on rules, logical optimization applies some optimization rules to the input logical execution plan in order, to make the whole logical execution plan better. The optimization rules include:
+
+- Column pruning
+- Eliminate projection
+- Decorrelate correlated subqueries
+- Eliminate Max/Min
+- Push down predicates
+- Partition pruning
+- Push down TopN and Limit
+
+## Physical optimization
+
+Based on cost, physical optimization makes the physical execution plan for the logical execution plan generated in the previous phase.
+
+In this phase, the optimizer selects the specific physical implementation for each operator in the logical execution plan. Different physical implementations of logical operators differs in time complexity, resource consumption, physical properties, and so on. During this process, the optimizer determines the cost of different physical implementations according to data statistics, and selects the physical execution plan with the minimum whole cost.
```

Comment:
... Different physical implementations of logical operators **differ** in time complexity, resource consumption, physical properties...